CMakeLists.txt

```diff
@@ -127,6 +150,26 @@ else()
   INCLUDE_DIRECTORIES(${SDL2_INCLUDE_DIRS} ${SDL2IMAGE_INCLUDE_DIRS})
 endif()
 
+if (OPENAL_ENABLED)
+  find_package(OpenAL)
+  find_package(ALUT)
+  if(TREMOR_ENABLED)
+    find_package(TREMOR)
+  else()
+    find_package(OGGVORBIS)
+  endif()
+  if (OPENAL_FOUND AND ALUT_FOUND AND ((TREMOR_ENABLED AND TREMOR_FOUND) OR ((NOT TREMOR_ENABLED) AND OGGVORBIS_FOUND)))
+    INCLUDE_DIRECTORIES(${OPENAL_INCLUDE_DIR} ${ALUT_INCLUDE_DIR})
+    if(TREMOR_ENABLED)
+      set(TREMOR 1)
+      INCLUDE_DIRECTORIES(${TREMOR_INCLUDE_DIR})
+    else()
+      INCLUDE_DIRECTORIES(${OGGVORBIS_INCLUDE_DIR})
```

Comment:
This doesn't work for me under OS X. It seems that `OGGVORBIS` is never defined. Rather, two separate variables are:
`VORBIS_INCLUDE_DIR` and `OGG_INCLUDE_DIR`
Including those two directories fixes the issue for me (or, rather, including only one is required since they're both located in the same directory).
If I had to take a guess, this works just fine on my Linux machine since their include directory is already included by practically every other library, thus hiding this issue.